samples/tab-deeplink/nodejs/pages/ChannelDeepLink.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title></title>
+        <script src='https://code.jquery.com/jquery-1.11.3.min.js'></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Could you please remove these script references are those are not used? Please check other html files and only keep the references which are being used.

Reply:
Updated HTML files and removed unnecessary script tags